script.js

```diff
@@ -51,29 +53,24 @@ buttonElement.addEventListener('click', function (event) {
 });
 
 buttonRight.addEventListener('click', function (){
-    move('right');
+    moveCarouselSlides('right');
 }); 
 buttonLeft.addEventListener('click', function (){
-    move('left');
+    moveCarouselSlides('left');
 });
 
-let images = document.querySelectorAll('.carousel object');
+let images = carouselBox.children;
 let slider = [];
-for (let i = 0; i < images.length; i++) {
+for (let i = images.length-1; i >= 0; i--) {
```

Comment:
Why not use way more efficient forEach or map functionality?

Reply:
Maybe. I should try.

Reply:
Can change to forEach